### PR TITLE
Issue #12092: Release notes for GitHub shows escape at-symbol

### LINKED ISSUE #12092: Release notes for GitHub shows escape at-symbol
https://github.com/checkstyle/checkstyle/pull/12007#issuecomment-1221431907

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Fixes #12092

### CODE REVIEW

File: .ci/update-github-page.sh [hunk 1]
```diff
@@ -67,9 +69,13 @@ java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.
 echo ==============================================
 echo "GITHUB PAGE:"
 echo ==============================================
-CONTENT=$(cat github_post.txt)
+CONTENT=$(cat "$FILE")
 echo "$CONTENT"
 
+if [[ -n $(grep "@" "$FILE" ) ]]; then
+  sed -i "s/@/"'`@`'"/g" "$FILE"
```
Comment (romani) [inline .ci/update-github-page.sh:76]: we need to escape content in JAVA code of releasenotes builder, to generate content that is ready for use without any hack after generation.

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: .ci/update-github-page.sh (hunk 1 above)
Comment (Rahulkhinchi03) [reply .ci/update-github-page.sh:76]: Okay, I am looking into it!

Comment (romani) [reply .ci/update-github-page.sh:76]: example of escaping in XML:
```
 <p>Notes:</p>
        <ul>
          <li>
            Github action &quot;Publish release notes outside&quot;.
            Author: github-actions[bot]
            <a href="https://github.com/checkstyle/checkstyle/issues/12082">#12082</a>
          </li>
          <li>
```
see it at https://cloud.drone.io/checkstyle/checkstyle/12325/4/2

**Comment by romani** [comment]
fix should be in releasenotes builder

**Pull request closed without merging**